### PR TITLE
Add congress_bill to database schema

### PR DESCRIPTION
**Pull request by kevincianfarini**

### CODE REVIEW

File: backend/src/main/kotlin/org/climatechangemakers/act/feature/bill/model/BillType.kt [hunk 1]
```diff
@@ -1,29 +1,18 @@
...
-import nl.adaptivity.xmlutil.serialization.XmlSerialName
 import org.climatechangemakers.act.common.serializers.StringEnum
 import org.climatechangemakers.act.common.serializers.StringEnumSerializer
 
-@Serializable class Bill(
-  @XmlElement(true) @XmlSerialName("BillCongress", "", "") val congressNumber: Int,
-  @XmlElement(true) @XmlSerialName("BillTypeAbbreviation", "", "") val billType: BillType,
-  @XmlElement(true) @XmlSerialName("BillNumber", "", "") val billNumber: Int,
-)
-
```
Comment (kevincianfarini) [inline backend/src/main/kotlin/org/climatechangemakers/act/feature/bill/model/BillType.kt:14]: This is no longer exclusive to CWC, and therefore we've lifted it form that namespace.

File: backend/src/main/kotlin/org/climatechangemakers/act/feature/communicatewithcongress/serialization/CommunicatingWithCongressBillSerializer.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.encoding.encodeStructure
+import org.climatechangemakers.act.feature.bill.model.Bill
+import org.climatechangemakers.act.feature.bill.model.BillType
+import kotlinx.serialization.descriptors.element
+import nl.adaptivity.xmlutil.serialization.XmlElement
+import nl.adaptivity.xmlutil.serialization.XmlSerialName
+
+object CommunicatingWithCongressBillSerializer : KSerializer<Bill> {
```
Comment (kevincianfarini) [inline backend/src/main/kotlin/org/climatechangemakers/act/feature/communicatewithcongress/serialization/CommunicatingWithCongressBillSerializer.kt:15]: I feel like there has to be a better solution to this, but I couldn't fine one other than hand writing the serializer. Normally this code is generated by the compiler which is why it's so ugly.

**Comment by kevincianfarini** [comment]
Gonna eagerly merge!

**Review by EvanHahn — COMMENTED** [review]
Glanced at the diff and it looks reasonable to me!

**Pull request merged**